tools/OpenConsole.psm1

```diff
@@ -31,7 +31,7 @@ function Import-LocalModule
 
     if (-not (Test-Path "$modules_root\$Name")) {
         Write-Verbose "$Name not downloaded -- downloading now"
-        $module = Find-Module 'VSSetup'
+        $module = Find-Module '$Name'

```

Comment:
This won't work. Single quotes in PowerShell don't expand variables. Just use `$Name` without quotes. Won't matter if it has spaces or not given how PowerShell parses and evaluates tokens.

Reply:
oops -.-